subsys/net/l2/ethernet/ethernet_mgmt.c

```diff
@@ -128,6 +128,45 @@ static int ethernet_set_config(u32_t mgmt_request,
 	return api->set_config(net_if_get_device(iface), type, &config);
 }
 
+static int ethernet_get_config(u32_t mgmt_request,
```

Comment:
move that function before line 191
The untold rule here is: 
- first the function
- then the relevant NET_MGMT_REGISTER_REQUEST_HANDLER() macro call(s) relevant to that function

Reply:
Sure! That makes perfect sense